controllers/commentController.js

```diff
@@ -0,0 +1,19 @@
+const commentService = require("../services/commentService");
+const { catchAsync } = require("../utils/error");
+
+const createComment = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const productId = req.params.productid;
+  const { content, rate } = req.body;
+
+  if (!userId || !content || !rate || !productId) {
+    return res.status(400).json({ message: "KEY_ERROR" });
+  }
+
+  await commentService.createComment(userId, content, rate, productId);
+  return res.status(201).json({
+    message: "create_comment_success",
```

Comment:
```suggestion
    message: "SUCCESS",
```
위에서 `KEY_ERROR`라고 작성하셨으니 그 형식에 맞게 아래도 작성해 주세요.
모든 통일된 하나의 방식으로 관리하는 것이 도움이 됩니다.